src/platform.h.in

```diff
@@ -15,6 +15,7 @@
 #ifndef NCNN_PLATFORM_H
 #define NCNN_PLATFORM_H
 
+#cmakedefine01 NCNN_SIMPLEMATH
```

Comment:
Consider moving `NCNN_SIMPLEPATH` to the bottom, just like the order when `src/CMakeLists.txt`.